src/infrastructure/content/IServerPort.java

```diff
@@ -1,10 +1,20 @@
-package infrastructure.content;
-
-public interface IServerPort {
-	/*
-	 * the processing module calls this method to send port of board server to each new client
-	 * @param : port, which is the port of board server in int datatype
-	 */
-	void sendPort(int port);
-
+package infrastructure.content;
+
+/**
+ * This Interface is used for sending port of board server to content module.
+ * Moreover, that port value can be accessed too with a separate method. 
+ * @author Badal Kumar (111701008)
+ */
+public interface IServerPort {
+	/**
+	 * the processing module calls this method to send port of board server to each new client
+	 * @param port - This is the port of board server in int datatype
+	 */

```

Comment:
This comment is very misleading. In fact, this method is just a setter method.
In file `ServerPort.java`, I see this,
```java

        /**
	 * This method will save the port of the board server locally to send message over networking
	 */
	@Override
	public void sendPort(int port) {
		this.port = port;
	}
```

The comment should be changed to something appropriate, and the method should be renamed to `setPort`.

@himanshu2047, since you must be using it, please take note of this.

Reply:
Are you renaming it? @Rahul-R5

Reply:
Yes, it is void setPort(int port)